newlib/libc/sys/vita/include/netdb.h

```diff
@@ -112,6 +112,7 @@ struct addrinfo {
 #define NI_WITHSCOPEID      0x00000020
 
 struct hostent *gethostbyname(const char *name);
+struct hostent *gethostbyaddr(const void *__addr, socklen_t __len, int __type);
```

Comment:
```suggestion
struct hostent *gethostbyaddr(const void *addr, socklen_t len, int ype);
```
could you remove underscores?

Reply:
I'll do that